java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5803_java.html

```diff
@@ -0,0 +1,43 @@
+<p>@VisibleForTesting can be used to mark methods, fields and classes whose visibility restrictions have been relaxed more than necessary for the API
+to allow for easier unit testing.</p>
+<p>Access to such methods, fields and classes only possible thanks to this relaxed visibility is fine for test code, but it should be avoided in
+production code.</p>
+<p>Supported framework:</p>
+<p> * Guava</p>
+<pre>
```

Comment:
Wops, it seems to me that this HTML file is completely broken. did you double-check how it looks in SonarQube?

Reply:
looking into it. Looks like need to update the Jira and regenerate

Reply:
Updated